python/src/PointToFieldFunctionalChaosAlgorithm_doc.i.in

```diff
@@ -0,0 +1,274 @@
+and the projected modes sample :math:`\mat{Q}`,
+see :class:`~openturns.LeastSquaresExpansion` for details.
+
+.. math::
+
+    \tilde{g}(x) = \sum_{k=1}^{K_c} \beta_{\vect{\alpha}_k} \Psi_{\vect{\alpha}_k}(x)
+
+with :math:`K_c` the number of terms in the chaos decomposition.
+
+The final metamodel consists in the composition of the functional chaos metamodel
+and the Karhunen-Loeve projections.
+
+.. math::
+    \tilde{h}: \left|
+      \begin{array}{ccccl}
+         \Rset^p & \rightarrow & \Rset^{K_T} & \rightarrow & \cM_N \times (\Rset^d)^N\\
+         \mat{X} & \mapsto & \vect{Q} & \mapsto & \vect{Y}
+      \end{array}
+    \right.
+
+Here the projected modes sample has a dimension :math:`K_T` but being on the output side
+of the functional chaos this approach is not affected by the curse of dimensionality,
+it will just require as much chaos marginals runs.
+
+Parameters
+----------
+x : :class:`~openturns.Sample`
+    Input sample.
+y : :class:`~openturns.ProcessSample`
+    Output process sample.
+distribution : :class:`~openturns.Distribution`
+    Input distribution
+
+See Also
+--------
+openturns.experimental.FieldToPointFunctionalChaosAlgorithm
+openturns.experimental.FieldFunctionalChaosSobolIndices
+openturns.KarhunenLoeveAlgorithm
+openturns.FunctionalChaosAlgorithm
+
+Notes
+-----
+As the output process decomposition is done with the values decomposition approach,
+it is possible to control the number of modes retained per output,
+the idea being to avoid a large output dimension for the chaos decomposition step.
+As done in :class:`~openturns.KarhunenLoeveSVDAlgorithm`, the :func:`setThreshold`
+and :func:`setNbModes` methods allow one to control the spectrum ratio and maximum count.
+
+In the case of homogenous data (if variables have the same unit or scale),
+it is also possible to recompress the modes at the global level with :func:`setRecompress`.
+When enabled, the eigenvalues are gathered and sorted so as to find a global
+spectrum cut-off value by which the spectrum of each output is truncated.
+The default value can be set through the :class:`~openturns.ResourceMap` key
+*PointToFieldFunctionalChaosAlgorithm-DefaultRecompress*.
+
+For the chaos metamodel step, it is possible to specify the basis size with
+the :class:`~openturns.ResourceMap` key *FunctionalChaosAlgorithm-BasisSize*.
+
+It is possible to separate output variables by independent blocks with :func:`setBlockIndices`,
+this way Karhunen-Loeve process decompositions are done on each block rather
+than on all output variables at once (at the price of increased chaos decomposition output dimension)
+which then makes sensitivity analysis possible for each output variable
+or group of output variables.
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> ot.RandomGenerator.SetSeed(0)
+>>> mesh = ot.RegularGrid(0.0, 0.1, 20)
+>>> X = ot.Normal(4)
```

Comment:
This falls into the PEP8 "names to avoid":

https://peps.python.org/pep-0008/#names-to-avoid

```suggestion
>>> inputDistribution = ot.Normal(4)
```

Reply:
this only applies to O and I to avoid confusion with other symbols but other single letter symbols are fine

Reply:
You are right: i misread the text. Sorry for the comment.